src/general/index.ts

```diff
@@ -110,3 +110,121 @@ export function getSelectedTabIndex(tabs: any[], selectedTabName: string): numbe
     return defaultIndex;
   }
 }
+
+
+/**
+ * This function can download a file on user's machine either directly by a url or a blob object.
+ *
+ * @param {
+ *  file:File | null;
```

Comment:
Are these params showing properly in the documentation html file?

Reply:
![image](https://user-images.githubusercontent.com/39800707/151750453-1f6c62ca-0c28-4abf-a6ee-0433ff420ed4.png)